packages/docusaurus/src/webpack/plugins/CleanWebpackPlugin.ts

```diff
@@ -152,6 +166,12 @@ export default class CleanWebpackPlugin {
       return;
     }
 
+    if (this.checkIfFileNameIsDuplicate()) {
+      throw new Error(
+        `Output file ${this.outputPath} already exists. Docusaurus needs this directory to save the build output. Either remove/change the file or choose a different build directory via '--out-dir'.`,
```

Comment:
```suggestion
        `A file at '${this.outputPath}' already exists. Docusaurus needs this path to be a directory to generate the static build output. Either remove/change the file or choose a different build directory via '--out-dir'.`,
```

"Output file build already exists" is not a good error message IMHO

There is no concept of "output file" in Docusaurus.